0-ad-history-mod-package/simulation/templates/units/han/champion_cavalry_spearman_academy.xml

```diff
@@ -8,6 +8,13 @@
     <Requirements>
       <Techs datatype="tokens">-unlock_champion_cavalry phase_city</Techs>
     </Requirements>
+    <History>
+        The Tiger and Leopard Cavalry was an elite cavalry division belonging to the warlord Cao Cao, commanded by his cousin Cao Chu. At that time, during the late Han dynasty and the early Three Kingdoms era (the first and second centuries BC), the use of cavalry was generally declining and shifting more towards heavier armor. There were other, similar cavalry divisions to the Tiger and Leopard cavalry in the armies of other warlords as well.
+        Admission into this special force had high requirements; applicants, for example, had to command at least a hundred units to be eligible. Because of that, both the commander, Cao Chun, and the squadron itself were renown for their elite status, extreme discipline, and physical capabilities. They could, for example, travel great distances in a comparably short amount of time.
```

Comment:
```suggestion
        Admission into this special force had high requirements; applicants, for example, had to command at least a hundred units to be eligible. Because of that, both the commander, Cao Chun, and the squadron itself were renown for their extreme discipline and physical capabilities. And could travel great distances in a comparably short amount of time.
```